### PR TITLE
IP 8

### PR DESCRIPTION
**Pull request by maijapapaya**
@jaredh159 @Henderjay

### CODE REVIEW

File: unabridged-works-volume-1/original/08-to-the-parlaiment.adoc [hunk 1]
```diff
@@ -4,8 +4,8 @@
 [.section-date]
 1659
 
-THAT there hath been a backsliding and turning aside from
-the GOOD OLD CAUSE even by the army (who formerly were glorious
+That there hath been a backsliding and turning aside from
+the _Good Old Cause_ even by the army (who formerly were glorious
```
Comment (maijapapaya) [inline unabridged-works-volume-1/original/08-to-the-parlaiment.adoc:8]: This was uncapitalized and italicized further down in the paragraph.

File: unabridged-works-volume-1/original/08-to-the-parlaiment.adoc [hunk 2]
```diff
@@ -204,10 +213,12 @@ and he will at length bring forth an instrument, which his soul can take pleasur
 and which he will make faithful to him in his work.
 O nation of England, wait on the Lord! wait on the Lord, O my soul!
 
+[.signed-section-context-close]
 From Chalfont in Bucks, the 18. of the 3. mon.
 1659.
 
 This from one who is a friend to the Commonwealth, a lover of true Freedom,
```
Comment (maijapapaya) [inline unabridged-works-volume-1/original/08-to-the-parlaiment.adoc:220]: AHHHH!!! Not sure what to do with this. I tried several different things and they all looked funny to me. I'm sure we've had stuff like this before, but I just can't remember how to work it.

File: unabridged-works-volume-1/original/08-to-the-parlaiment.adoc (hunk 1 above)
Comment (jaredh159) [reply unabridged-works-volume-1/original/08-to-the-parlaiment.adoc:8]: I like how you did it here, let's go with it. 👍

File: unabridged-works-volume-1/original/08-to-the-parlaiment.adoc (hunk 2 above)
Comment (jaredh159) [reply unabridged-works-volume-1/original/08-to-the-parlaiment.adoc:220]: Hmmm... my gut is to try a [.postscript]. ...I just tried it locally and I think that it looks good too.  Let's change it to:

```ASCIIDOC
[.postscript]
====

This from one who is a friend to the Commonwealth, a lover of true Freedom,
and desires the good of all men.

====
```

**Comment by jaredh159** [comment]
Thanks, merged! 👍

**Pull request merged**